refinery/core/search_indexes.py

```diff
@@ -45,8 +45,16 @@ def index_queryset(self, using=None):
         """Used when the entire index for model is updated"""
         return self.get_model().objects.all()
 
-    def prepare_description(self, object):
-        return object.get_investigation().get_description()
+    def prepare_description(self, object, description=""):
+        try:
+            description = object.get_investigation().get_description()
+        except AttributeError as e:
+            logger.error(
+                "Could not fetch Investigation for DataSet with UUID: %s %s",
+                object.uuid,
+                e
+            )
+        return description
```

Comment:
Adding unnecessary arguments to function signatures should be avoided. One simple solution is to put `return object.get_investigation().get_description()` in the try block and add `return ""` at the very end.